sulu/form-bundle/2.0/config/packages/sulu_form.yaml

```diff
@@ -0,0 +1,9 @@
+sulu_form:
+    mail:
+        from: "%env(SULU_ADMIN_EMAIL)%"
+        to: "%env(SULU_ADMIN_EMAIL)%"
+        sender: "%env(SULU_ADMIN_EMAIL)%"
+
+framework:
+    esi: true
```

Comment:
Not happy with this but currently needed. Did create an issue for it on our side: https://github.com/sulu/SuluFormBundle/issues/265

Reply:
👍

Reply:
I agree. Im not happy with this either. 

Thank you for the issue. We should make sure these are removed when https://github.com/sulu/SuluFormBundle/issues/265 is fixed.